### ISSUE TITLE
Permintaan fitur: Penduduk Luar Desa

### ISSUE THREAD

**Issue by alit2310** [OpenSID/OpenSID#8306]
### Apa kekurangan yg ingin diatasi?

Penduduk Desa dan penduduk non permanen (tidak tetap) masih tergabung menjadi satu. Di tampilan Front End web jumlah penduduk desa dan penduduk non permanen dijumlah menjadi satu, sehingga mengakibatkan lonjakan penduduk tinggi.

### Jelaskan solusi yg diinginkan

Buatkan tampilan khusus untuk penduduk non permanen dan jumlahnya jangan digabung dengan penduduk Desa
![test](https://github.com/user-attachments/assets/3655bee8-51df-40bf-801a-4284e20d6483)

### Apakah ada alternatif lain

Buatkan tampilan khusus untuk penduduk non permanen dan jumlahnya jangan digabung dengan penduduk Desa

### Untuk versi OpenSID mana

24.08.1.0

**Comment by bagusmahendra85** [comment]
setuju, lebih tepat jika penduduk tetap dan non-permanen dipisah jumlahnya
